src/helpers/validation.ts

```diff
@@ -1,14 +1,5 @@
-import Ajv from 'ajv';
 import type { ErrorObject } from 'ajv';
```

Comment:
Do we really need this `ajv` dependency if we only use it for the type ?

Reply:
Removed it 👍